runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/RuntimeContext.kt

```diff
@@ -0,0 +1,42 @@
+package io.openruntimes.kotlin
+
+import kotlin.Any
+import kotlin.collections.ArrayList
+import kotlin.collections.arrayListOf
+
+import com.google.gson.GsonBuilder
+import com.google.gson.Gson
+
+class RuntimeContext(req: RuntimeRequest, res: RuntimeResponse) {
+    var req: RuntimeRequest
+    var res: RuntimeResponse
```

Comment:
You can use constructor promotion here, and use `val` for immutable variables:

```suggestion
class RuntimeContext(val req: RuntimeRequest, val res: RuntimeResponse) {
```